typings/index.d.ts

```diff
@@ -0,0 +1,133 @@
+import {
+    CSSProperties,
+    CSSPropertiesArray,
+    CSSPropertiesComplete,
+    CSSPropertiesLossy,
+    CSSPropertiesPseudo,
+    CSSPropertiesRecursive,
+    CSSWideKeyword,
+} from './css-properties';
+
+export {
+    CSSProperties,
+    CSSPropertiesArray,
+    CSSPropertiesComplete,
+    CSSPropertiesLossy,
+    CSSPropertiesPseudo,
+    CSSPropertiesRecursive,
+    CSSWideKeyword,
+};
+
+/**
+ * Aphrodite style declaration
+ */
+export type StyleDeclarationMap = Map<keyof CSSProperties, string | number>;
+export type StyleDeclaration<T = {}> = {
+    [P in keyof T]: CSSProperties | StyleDeclarationMap
+};
+
+/**
+ * Return value from StyleSheet.create.
+ */
+export interface StyleDeclarationValue<
+    T extends CSSProperties | StyleDeclarationMap = {}
+> {
+    _len: number;

```

Comment:
Since this is part of the private API, I don't know if it's valuable to export these properties, since this is intentionally not part of the public API.

Reply:
That sounds good to me :)